modules/consul/src/test/java/org/testcontainers/consul/ConsulContainerTest.java

```diff
@@ -31,21 +30,19 @@ public class ConsulContainerTest {
     public void readFirstPropertyPathWithCli() throws IOException, InterruptedException {
         GenericContainer.ExecResult result = consulContainer.execInContainer("consul", "kv", "get", "config/testing1");
         final String output = result.getStdout().replaceAll("\\r?\\n", "");
-        MatcherAssert.assertThat(output, CoreMatchers.containsString("value123"));
+        assertThat(output).contains("value123");
     }
 
     @Test
-    public void readFirstSecretPathOverHttpApi() throws InterruptedException {
-        RestAssured
+    public void readFirstSecretPathOverHttpApi() {
+        io.restassured.response.Response response = RestAssured
```

Comment:
Importing `io.restassured.response.Response`, or?

Reply:
there is already a Response imported from consul java library